cha/usecase/vol_area.tex

```diff
@@ -0,0 +1,54 @@
+\subsection{Increased volume and area}
+This category of use cases have the thing in common that it uses the synchronization to increase the volume or area of the music.
+Some of the aspects of the use cases occur in several of them, for instance the requirement of precise synchronization.
+
+Examples of use cases in this category could be the following:
+
+\begin{description}
+    \item[Social gathering/Party] \hfill\\
+        At a social gathering or party, dependent on the event, the participants wants to be able to listen to loud music.
+        If no dedicated sound system is available, phones can be used. 
+        Phones have a low music volume and if some participants are talking and dancing, it can be hard to hear the music.
+
+        With synchronization of the music, the participants can use their phone and play the same music as one big speaker.
+        This will make the perceived volume of the music louder to the listeners and the overall experience of listening to the music better.
+        If some at the party or gathering just want to talk, they can either turn down the volume of their devices, mute the music, or turn the music off,
+        making it more quieter for them while the other participants still can hear the music.
+        For all the participants to have the impression of only one speaker is playing, 
+        the synchronization have to be precise enough so that there is no difference in playback of the music between the devices.
+
+    \item[Festival scenario] \hfill\\
+        At a music festival, there is often an area for the concerts and an area for people to sleep, often called a camp.
+        These camps can stretch over a large area, dependent on the size of the festival.
+        At the camp there is often party before the concerts starts or after it has ended.
+        It is difficult to play music over the whole camp and maybe not all want to hear the music at the same volume.
+
+        Many, if not all, participants have phones which can be used to play music.
+        By having these phones connect to a device, which can send synchronized music to the phones,
+        the festival participants can all play the same synchronized music,
+        increasing the area covered by the music.
+        As with the social gathering/party use case,
+        the festival participants should be able to control the volume locally without affecting the rest.
+
+    \item[Multi-room setup] \hfill\\
+        When at home, people often listen to music.
+        In the case that they live in a big apartment or house, they can only hear the music in one room,

```

Comment:
Jeg tror det ville være bedre at frame det som en drøm:
"What if the music playback would be synchronous across all rooms of the home?" 

Eller sådan noget